src/use-cases/enrichHearingOutcome/enrichFunctions/enrichCase.test.ts

```diff
@@ -0,0 +1,17 @@
+import { readFileSync } from "fs"
+import parseSpiResult from "src/use-cases/parseSpiResult"
+import enrichCase from "./enrichCase"
+import transformSpiToAho from "../../transformSpiToAho"
+
+const message = readFileSync("test-data/input-message-001.xml", "utf-8")
+const spiResult = parseSpiResult(message)
+const aho = transformSpiToAho(spiResult)
+
+describe("enrich case", () => {
+  it("should return AHO enriched with case", () => {
+    const result = enrichCase(aho)
```

Comment:
Can we add some more specific tests for the actual data that this function enriches with? Snapshots are useful for detrecting change but they can be too easy to update.

Reply:
@bjpirt added some tests in [08f8f56](https://github.com/ministryofjustice/bichard7-next-core/pull/48/commits/08f8f56771eb1a02489b5c4cd1bc52a39be2078e)